modules/client-configuration/pages/clients-almalinux.adoc

```diff
@@ -148,14 +148,27 @@ For more information on activation keys, see xref:client-configuration:activatio
 
 
 
-ifeval::[{uyuni-content} == true]
+//ifeval::[{uyuni-content} == true]
 
-== Trust GPG Keys on Clients
+//== Trust GPG Keys on Clients
 
-include::snippets/trust_gpg.adoc[]
+//include::snippets/trust_gpg.adoc[]
 
-endif::[]
+//endif::[]
+
+== Manage GPG Keys
+
+Clients use GPG keys to check the authenticity of software packages before they are installed. 
+Only trusted software can be installed on clients.
+
+[IMPORTANT]
+====
+Trusting a GPG key is important for security on clients.
+It is the task of the adminstrator to decide which keys are needed and can be trusted.
+Because a software channel cannot be used when the GPG key is not trusted, the decision of assigning a channel to a client depends on the decision of trusting the key.
```

Comment:
I proposed an accepted phrase in the old or existing `modules/client-configuration/pages/snippets/trust_gpg.adoc` file.  I'd appreciate if we could use this wording.  Also in `master` we probably should use this wording:
> It is the task of the administrator to decide which keys are needed and can be trusted.
> A software channel cannot be assigned to a client when the GPG key is not trusted.